Header header
float32 threshold
float32[] signal_derivative
int32[] peaks_index
